### PR TITLE
fix(config): separate Neo CoolCam NAS-WR01ZE V2 from WR01Z

### PR DESCRIPTION
**Pull request by Masterz69**
Based on information in issue #1221 about NAS-WR01ZE firmware V2, as well as checked with my owned plug manuals.

### CODE REVIEW

**Comment by AlCalzone** [comment]
I'm a bit confused - are the WR01Z and WR01ZE different devices?

**Comment by Masterz69** [comment]
As per openzwave.com - see links in #1221:

   <MetaDataItem id="1027" name="Identifier" type="0200">NAS-WR01ZE</MetaDataItem>
    <MetaDataItem id="1027" name="Identifier" type="0100">NAS-WR01ZE</MetaDataItem>
    <MetaDataItem id="1027" name="FrequencyName" type="0100">U.S. / Canada / Mexico</MetaDataItem>
    <ChangeLog>
      <Entry author="David Burrows - david@burrows.org" date="19 Aug 2020" revision="2">Added metadata, and added US version (type 0100) that is branded "Power Plug" with no model number displayed. There is no matching Z-Wave Alliance database entry.</Entry>
    </ChangeLog>


0258:1027:0200 comparing with 0258:0087:0003 - parameters in different order.

**Comment by AlCalzone** [comment]
I don't trust the OZW database anymore. So these are two different devices with just a very similar name?
Google search doesn't help much.

**Comment by Masterz69** [comment]
Sure, see no clear picture too.

Me and @jghaanstra have on hands devices 0258:1027:0200.
They have same parameter as 0258:0087:0003, but in different order.

ZWave alliance database contains https://products.z-wavealliance.org/products/1783
^^ NAS-WR01Z == 0258:0087:0003 - w/out E letter.

And no ZE model in products.z-wavealliance.org 
- may be Chinese manufacturing something with finished certification/approval ?

**Comment by AlCalzone** [comment]
Ok we can treat the ZE as the new revision then. Makes sense.

I'll review the change itself later, will let you know.

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+			"minValue": 1,
+			"maxValue": 15,
+			"defaultValue": 12,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "LED enable",
+			"description": "This parameter defines the LED indication Function ON/OFF.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"label": "LED Indication",
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+			"options": [
+				{
+					"label": "Disable LED indication Function",
+					"value": 0
+				},
+				{
+					"label": "Enable LED indication Function",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
			]
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+		},
+		"6": {
+			"label": "Power Report Threshold",
+			"description": "Power consumed change to be reported",
+			"unit": "percentage",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"unit": "%",
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+				},
+				{
+					"label": "Enable LED indication Function",
+					"value": 1
+				}
+			]
+		},
+		"6": {
+			"label": "Power Report Threshold",
+			"description": "Power consumed change to be reported",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"description": "Change in power consumption that triggers a report",
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+			"minValue": 1,
+			"maxValue": 100,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"7": {
+			"label": "Remember Relay ON/OFF status after power failure.",
+			"description": "Plug state after power-on",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"label": "Remember Relay ON/OFF status after power failure",
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+			"options": [
+				{
+					"label": "Don’t remember relay state",
+					"value": 0
+				},
+				{
+					"label": "Remember the Relay Status.",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"options": [
				{
					"label": "Do not remember relay state",
					"value": 0
				},
				{
					"label": "Remember relay state",
					"value": 1
				}
			]
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+		},
+		"8": {
+			"label": "Plug Time switch Function",
+			"description": "Timer plug OFF after time period",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"label": "Timed auto-off",
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+			"options": [
+				{
+					"label": "Time switch Disable",
+					"value": 0
+				},
+				{
+					"label": "Time switch Enable",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
			]
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+		},
+		"9": {
+			"label": "Time switch Period",
+			"description": "Delay time after the plug is switched off (in minutes)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"label": "Timed auto-off period",
			"description": "Delay time after which the plug is automatically switched off",
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+			"unit": "Minutes",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"unit": "minutes",
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+			"maxValue": 65535,
+			"defaultValue": 150,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "Button On/Off Enable",
+			"description": "Enable/Disable top button function",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"label": "Top button function",
```

File: packages/config/config/devices/0x0258/nas-wr01z.json [hunk 12]
```diff
@@ -0,0 +1,200 @@
...
+			"options": [
+				{
+					"label": "Disable Button",
+					"value": 0
+				},
+				{
+					"label": "Enable Button",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01z.json]:
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
			]
```

File: packages/config/config/devices/0x0258/nas-wr01ze.json [hunk 13]
```diff
@@ -34,8 +22,8 @@
 	"supportsZWavePlus": true,
 	"paramInformation": {
 		"1": {
-			"label": "Meter Report Enable",
-			"description": "This parameter defines Disable/Enable meter report measure data to controller.",
+			"label": "Remember Relay ON/OFF status after power failure.",
+			"description": "Plug state after power-on",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01ze.json]: see previous file

File: packages/config/config/devices/0x0258/nas-wr01ze.json [hunk 14]
```diff
@@ -45,55 +33,18 @@
...
 			"options": [
 				{
-					"label": "Disable Report",
+					"label": "Don’t remember relay state",
 					"value": 0
 				},
 				{
-					"label": "Enable Report",
+					"label": "Remember the Relay Status.",
 					"value": 1
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01ze.json]: see previous file

File: packages/config/config/devices/0x0258/nas-wr01ze.json [hunk 15]
```diff
@@ -45,55 +33,18 @@
...
-			"defaultValue": 12,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
-		},
-		"5": {
-			"label": "LED enable",
-			"description": "This parameter defines the LED indication Function ON/OFF.",
+			"label": "Button On/Off Enable",
+			"description": "Enable/Disable top button function",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01ze.json]: see previous file

File: packages/config/config/devices/0x0258/nas-wr01ze.json [hunk 16]
```diff
@@ -103,30 +54,18 @@
...
 				{
-					"label": "Disable LED indication Function",
+					"label": "Disable Button",
 					"value": 0
 				},
 				{
-					"label": "Enable LED indication Function",
+					"label": "Enable Button",
 					"value": 1
 				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01ze.json]: see previous file

Comment (AlCalzone) [reply packages/config/config/devices/0x0258/nas-wr01ze.json]: same for all the other parameters that exist in both files

File: packages/config/config/devices/0x0258/nas-wr01ze.json [hunk 17]
```diff
@@ -191,14 +130,63 @@
...
+			"maxValue": 16,
+			"defaultValue": 16,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"9": {
+			"label": "Power Report Threshold",
+			"description": "Power consumed change to be reported",
+			"unit": "cA",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0258/nas-wr01ze.json]: What kind of unit is this? I guess `"0.01 A"` which we prefer because it is obvious.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by blhoward2** [comment]
@Masterz69 Earlier today we merged detailed documentation on device files, including a style guide and detailed guidance on templating. Could you please take a look at those, make any changes necessary to conform, and then let us know when you're ready for us to take a look line-by-line?

https://zwave-js.github.io/node-zwave-js/#/

**Review by AlCalzone — CHANGES_REQUESTED** [review]
What @blhoward2 said :)

**Comment by Masterz69** [comment]
Both files styling adjustment - label, descriptions & units.

**Comment by Masterz69** [comment]
@zwave-js-bot fix lint

**Review by AlCalzone — APPROVED** [review]
Thanks!

**Pull request merged**